prov/gni/src/gnix_vc.c

```diff
@@ -451,10 +378,12 @@ static int __gnix_vc_hndl_wc_match_con(struct gnix_datagram *dgram,
 	 */
 	assert(wc_vc->conn_state == GNIX_VC_CONNECTING);
 
+#if 0
 	/*
 	 * remove the vc from the EPs dlist of wc vc's
 	 */
 	dlist_remove(&wc_vc->entry);
+#endif
```

Comment:
I should remove this code.  This is commented out because we need a way to destroy all of the vc's being used by an FI_EP_RDM when its being closed.  We don't currently have a way to easily destroy the vc's at job termination using the hash table - we might want to talk about this tomorrow or Thursday.  Note that for the special case of the wc_vc not being in the hash table, just below, it is removed from the linked list.

This linked list is the way currently being used to cleanup vc's in ```gnix_ep_close```  method.